lib/serialize.js

```diff
@@ -115,15 +116,14 @@ module.exports = function (app, defaults) {
       app: app,
       model: model,
       method: ctx.method.name,
+      meta: ctx.meta ? utils.clone(ctx.meta) : null,
```

Comment:
In my use case I only need to add the meta tag in the afterJsonApiSerialize - but iirc then I had access to all the same objects throughout that chain.

Re: adding the meta tag per item... that sounds like it would be useful, but is liable to push out the time taken to get this PR wrapped up.